homeassistant/components/rtsptowebrtc/manifest.json

```diff
@@ -0,0 +1,12 @@
+{
+  "domain": "rtsptowebrtc",
```

Comment:
Our standard is to separate words with underscore in the domain name. Maybe change to `rtsp_to_webrtc`?

Reply:
Done.